src/components/GameConsole.tsx

```diff
@@ -57,7 +57,6 @@ export default function GameConsole() {
         'Content-Type': 'application/json',
       },
     });
-    scrollToBottom();
```

Comment:
I think this makes sense?
When you send a new command it should scroll to bottom